README.md

```diff
@@ -44,27 +44,33 @@ wifi.setmode(wifi.STATION)
 wifi.sta.config{ssid="SSID", pwd="password"}
 ```
 
-# Documentation
+## Documentation
 
 The entire [NodeMCU documentation](https://nodemcu.readthedocs.io) is maintained right in this repository at [/docs](docs). The fact that the API documentation is maintained in the same repository as the code that *provides* the API ensures consistency between the two. With every commit the documentation is rebuilt by Read the Docs and thus transformed from terse Markdown into a nicely browsable HTML site at [https://nodemcu.readthedocs.io](https://nodemcu.readthedocs.io).
 
+
+Pages:
+
 - How to [build the firmware](https://nodemcu.readthedocs.io/en/release/build/)
 - How to [flash the firmware](https://nodemcu.readthedocs.io/en/release/flash/)
 - How to [upload code and NodeMCU IDEs](https://nodemcu.readthedocs.io/en/release/upload/)
 - API documentation for every module
 
-# Releases
+
+## Releases
 
 Due to the ever-growing number of modules available within NodeMCU, pre-built binaries are no longer made available. Use the automated [custom firmware build service](http://nodemcu-build.com/) to get the specific firmware configuration you need, or consult the [documentation](http://nodemcu.readthedocs.io/en/release/build/) for other options to build your own firmware.
 
 This project uses two main branches, `release` and `dev`. `dev` is actively worked on and it's also where PRs should be created against. `release` thus can be considered "stable" even though there are no automated regression tests. The goal is to merge back to `release` roughly every 2 months. Depending on the current "heat" (issues, PRs) we accept changes to `dev` for 5-6 weeks and then hold back for 2-3 weeks before the next snap is completed.
 
-A new tag is created every time `dev` is merged back to `release`. They are listed in the [releases section here on GitHub](https://github.com/nodemcu/nodemcu-firmware/releases). Tag names follow the \<SDK-version\>-release_yyyymmdd pattern.
+A new tag is created every time the `dev` branch is merged back to `release`. They are listed in this repo's [releases](https://github.com/nodemcu/nodemcu-firmware/releases).
+
+Tag names follow the `<SDK-version>-release_yyyymmdd` pattern.
```

Comment:
This is on its own line otherwise when viewing the page on GitHub the codeblock wraps across to a new line in the middle of the line so it looks weird.